components/Carousel/Carousel.js

```diff
@@ -1,8 +1,16 @@
 class Carousel {
```

Comment:
There's a few ways to go about this, check out how I went about it and try to understand what's being done on each line: 
```
class Carousel {
 constructor(element) {
     this.carousel = element;
     this.rightBtn = this.carousel.querySelector('.right-button');
     this.leftBtn = this.carousel.querySelector('.left-button');
     this.carouselItems = this.carousel.querySelectorAll('img');

     this.counter = 0;
     this.imgTotal = Array.from(this.carouselItems).length;

     this.currentImg = this.carouselItems[0];
     this.currentImg.style.display = 'block';

     this.rightBtn.addEventListener('click', () => this.next());
     this.leftBtn.addEventListener('click', () => this.prev());
    }
     
    prev(){
        this.currentImg.style.display = 'none';
        if (this.counter === 0) {
            this.counter = this.imgTotal-1;
            this.currentImg = this.carouselItems[this.counter];
            this.currentImg.style.display = 'block';
        } else {
            this.counter -= 1;
            this.currentImg = this.carouselItems[this.counter];
            this.currentImg.style.display = 'block';
        }
    }

    next(){
        this.currentImg.style.display = 'none';
        if (this.counter === this.imgTotal-1) {
            this.counter = 0;
            this.currentImg = this.carouselItems[this.counter];
            this.currentImg.style.display = 'block';
        } else {
            this.counter += 1;
            this.currentImg = this.carouselItems[this.counter];            
            this.currentImg.style.display = 'block';
        }
    }
}

let carousel = new Carousel(document.querySelector('.carousel'));
```